src/server/endpoints/home-screen-endpoints.js

```diff
@@ -19,13 +19,17 @@ function getHomeInfos(req, res) {
     let userId;
 
     if (token === undefined) {
-      res.status(400).json({'error': 'unauthorization'});
```

Comment:
if the authorization header is missing it is a 400 bad request. 401 if we got a token but it is expired or we couldnt find in the db.